core/dvector.h

```diff
@@ -424,6 +425,17 @@ Error DVector<T>::resize(int p_size) {
 	return OK;
 }
 
+template<class T>
+void DVector<T>::invert() {
+	T temp;
+
+	for(int i=0;i<size()/2;i++) {
+		temp = get(i);
+		set(i, get(size()-i-1));
+		set(size()-i-1, temp);
+	}
```

Comment:
Yes, I should assign it to a local variable before the loop so I don't have to make more than one call to `size()`.